the_debuginator.h

```diff
@@ -1552,6 +1581,8 @@ void debuginator_update_filter(TheDebuginator* debuginator, const char* wanted_f
 		fallback->parent->folder.hot_child = fallback;
 	}
 
+	debuginator__set_item_total_height_recursively(debuginator->root, debuginator->item_height);
```

Comment:
Didn't see this before. debuginator__adjust_num_visible_children gets called for every item that gets filtered away. When you have lots of items it's potentially extremely expensive to do this. Do we need to do it on the root?